packages/config/config/devices/0x0346/motion_sensor_gen2.json

```
@@ -0,0 +1,195 @@
+// Ring - / 4SP1SZ-0EU0
+// Motion Sensor Gen 2
+{
+	"manufacturer": "Ring",
+	"manufacturerId": "0x0346",
+	"label": "- / 4SP1SZ-0EU0",
+	"description": "Motion Sensor Gen2",
+	"devices": [
+		{
+			"productType": "0x0301",
+			"productId": "0x0301"
+		},
+		{
+			"productType": "0x0301",
+			"productId": "0x0401"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Heartbeat Interval",
+			"valueSize": 1,
+			"unit": "minutes",
+			"minValue": 1,
+			"maxValue": 70,
+			"defaultValue": 70,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"label": "Application Retries",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 5,
+			"defaultValue": 5,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"3": {
+			"label": "Application Level Retries",
+			"valueSize": 1,
+			"unit": "seconds",
+			"minValue": 1,
+			"maxValue": 96,
+			"defaultValue": 5,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"4": {
+			"label": "Led Indicator",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Don't show green",
+					"value": 0
+				},
+				{
+					"label": "Show green after Fault",
+					"value": 1
+				},
+				{
+					"label": "Show green after Intrusion and Intrusion clear",
+					"value": 2
+				}
+			]
+		},
+		"5": {
+			"label": "Occupancy Clear Delay",
+			"valueSize": 1,
+			"unit": "seconds",
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"unsigned": true,
+			"allowManualEntry": true
+		},
+		"6": {
+			"label": "Instrusion Clear Delay",
+			"valueSize": 1,
+			"unit": "seconds",
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"unsigned": true,
+			"allowManualEntry": true
+		},
+		"7": {
+			"label": "Standard Clear Delay",
+			"valueSize": 1,
+			"unit": "seconds",
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"unsigned": true,
+			"allowManualEntry": true
+		},
+		"8": {
+			"label": "Motion Detection Mode",
+			"description": "Adjusts motion sensitivity",
```

```suggestion
			"label": "Motion Detection Sensitivity",
			"description": "0 = lowest, 4 = highest",
```
or whatever the values mean?

---

Manual doesn't say, I believe it's the pet immunity like the first one but couldn't find anything anywhere and didn't want to guess. On Gen1, it was only 3 options, no pet immunity, up to 15kg and up to 25 kg.